### ISSUE TITLE
Bad or missing accessibility labels for UI controls

### ISSUE THREAD

**Issue by pitermach** [owncloud/ios#771]
### Steps to reproduce
1. Enable VoiceOver screen reader, either with Siri or in settings>general>Accessibility>VoiceOver (note that this will change the way gestures work in the system as long as it's running)
2. Open OwnCloud app and explore the screen, either flicking left/right or by dragging finger around
3. Notice how screen items are announced, starting on the login screen (a few specific examples are below)

### Expected behaviour
VoiceOver should announce text descriptions for any items on the screen, usually with 1 or 2 words

### Actual behaviour
Most controls either have no label, or an overly descriptive one generated probably from its icon filename. So, most controls on the login screen are just announced as "Button" though thankfully there is text next to the important ones. While browsing files, every file has buttons  next to it like "more filled", "SharedItemSwipe". The shared links tab has a few controls with no label, etc. 

### Server configuration
**Web server:**

**Database:**

**PHP version:**

### Client
**iOS version:**
10 GM
**ownCloud app version:**
3.5.0
**Device model:**
iPhone 6
#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```
 
### Tell us what could be improved:
The controls just need accessibility labels set on them so that VoiceOver announces them correctly. For an even better experience, the buttons next to every file could be hidden to VoiceOver and exposed using the "actions" feature (you can look at apps like Dropbox or Onedrive to see how this can be done so that it is  pleasant to use). Apple has documentation on improving app accessibility here:
https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/iPhoneAccessibility/Introduction/Introduction.html
